### DISCUSSION TITLE
“translate page”-function when right click – Yes or no?

### DISCUSSION THREAD

**Discussion by Ransom77** [Eloston/ungoogled-chromium#3109] [General]
I have now jumped back and forth between many of the pages listed here to find out whether Google Chrome’s integrated translator is definitely not to be activated or whether this will probably never be the case.

I am only interested in this (very practical) function, not in alternatives to it.

Can anyone comment on this? Thank you.

**Comment by PF4Public** [comment]
You already answered your question

> to find out whether Google Chrome’s integrated translator is definitely not to be activated or whether this will probably never be the case.

**Comment by setanarut** [comment]
I recommend this extension, it is the only extension that works properly. 

https://chromewebstore.google.com/detail/twp-translate-beta/gkkkcomfmldkigajkmljnbpiajbpbgdg
